format/index.md

```diff
@@ -52,761 +58,507 @@ called out below.
-
-A "collection object" contains one or more of the members:
-
-* `"ids"` - an array of IDs for the referenced resources.
-* `"type"` - the resource type.
-* `"href"` - the URL of the referenced resources (applicable to response
-  documents).
-
-A server that provides a collection object that contains an `"href"` **MUST**
-respond to a `GET` request to the specified URL with a response that includes
-the referenced objects as a collection of resource objects.
-
-
-### URL Templates <a href="#document-structure-url-templates" id="document-structure-url-templates" class="headerlink"></a>
+A server **MUST** respond to a `GET` request to the specified URL with a
+response that includes the resource as the primary data.
 
-A top-level `"links"` object **MAY** be used to specify URL templates that can
-be used to formulate URLs for resources according to their type.
+#### Resource Relationships <a href="#document-structure-resource-relationships" id="document-structure-resource-relationships" class="headerlink"></a>
 
-For example:
+A resource object **MAY** contain references to other resource objects
+("relationships"). Relationships may be to-one or to-many. Relationships
+can be specified by including a member, keyed by the name of the
+relationship (anything but `"self"`), in a resource's links object.
+
+The value of a relationship **MUST** be one of the following:
+
+* A string, which represents a URL for the related resource(s) (a "related
+  resource URL"). When fetched, it returns the related resource object(s) as the
+  response's primary data. For example, an `article`'s `comments` could specify a
+  URL that returns a list of comment resource objects when retrieved through a
+  `GET` request. A related resource URL **SHOULD** remain constant even when the
+  resource(s) it represents mutate.
+
+* An object (a "link object").
+
+If a relationship is provided as a link object, it **MUST** contain at least
+one of the following:
+
+* A `self` member, whose value is a URL for the relationship itself (a
+  "relationship URL"). This URL allows the client to directly manipulate the
+  relationship. For example, it would allow a client to remove an `author` from
+  an `article` without deleting the `people` resource itself.
+* A `resource` member, whose value is a related resource URL (as defined above).
+* Linkage to other resource objects ("object linkage") included in a compound
+  document. This allows a client to link together all of the resource objects
+  included in a compound document without having to `GET` one of the
+  relationship URLs. Linkage **MUST** be expressed as:
+  * `type` and `id` members for to-one relationships.
+  * `type` and `ids` members for homogenous to-many relationships.
+  * A `data` member whose value is an array of objects each containing `type`
+    and `id` members for heterogenous to-many relationships.
+* A `"meta"` member that contains non-standard meta-information about the
```

Comment:
Is "meta" really one of the "at least one," or is it an additional **MAY** on top of at least one of the preceding three? In other words: could I include an object that *only* has a meta property?